docs/discovery/webfinger.md

```diff
@@ -2,88 +2,43 @@
 title: WebFinger
 ---
 
-diaspora\* uses an old draft of [WebFinger][webfinger-draft] to discover users from other pods.
+diaspora\* uses [WebFinger][webfinger-rfc] to discover users from other pods.
 
-{% include warning_box.html
-   title="Old WebFinger"
-   content="<p>diaspora* doesn't yet fully support the RFC 7033 WebFinger!</p>"
-%}
-
-## WebFinger endpoint discovery
-
-To find the WebFinger endpoint, the requesting server must get the [host metadata][host-meta] information for the
-domain of the searched diaspora\* ID.
+## WebFinger
 
 ### Request
 
-Let's assume we are searching for `alice@example.org`, then we need the host metadata for `example.org`.
-
-The request should first be tried with https, if this doesn't work, the requesting server should fallback to http.
-
-If the server response indicates that the host-meta resource is located elsewhere (a 301, 302, or 307 response status
-code), the requesting server should try to obtain the resource from the location provided in the response.
-
-#### Example
-
-~~~
-GET /.well-known/host-meta
-Host: example.org
-~~~
-
-### Response
-
-The host-meta document must be in the [XRD 1.0 document format][xrd] and should be served with the
-`application/xrd+xml` media type.
-
-#### Mandatory Link Relations
-
-The Host Metadata response must contain the following link relations:
-
-| Link Relation | Type         | Description                                                               |
-| ------------- | ------------ | ------------------------------------------------------------------------- |
-| lrdd          | url template | The template to the URL where the server provides the WebFinger endpoint. |
-
-The Host Metadata response may contain other optional link relations.
-
-#### Example
-
-~~~
-Status: 200 OK
-Content-Type: application/xrd+xml; charset=utf-8
 ~~~
-~~~xml
-<?xml version="1.0" encoding="UTF-8"?>
-<XRD xmlns="http://docs.oasis-open.org/ns/xri/xrd-1.0">
-  <Link rel="lrdd" type="application/xrd+xml" template="https://example.org/.well-known/webfinger.xml?resource={uri}"/>
-</XRD>
+GET /.well-known/webfinger
 ~~~
 
-## WebFinger
+Let's assume we are searching for `alice@example.org`, then we need to make a request to `example.org`.
 
-### Request
+The request should first be tried with https, if this doesn't work, the requesting server should fallback to http.
```

Comment:
This should be updated according to our decion upon http. We shouldn't say "the requesting server should fallback to http". Instead we can say that by default we support https only (since we follow the RFC 7033), but a protocol implementation can optionally support `http` schema fallback which is not RFC 7033 compliant, but can be useful in some cases.

Reply:
Good catch. I just removed the sentence completely. If it is in the RFC, we don't need to repeat it here. I only described the most basic WebFinger here and didn't copy/repeat the whole RFC. It's only to mention what parts in the response are required for us, everything else is in the RFC.

And I also don't think we need to allow people to break the RFC here. If people do that (for whatever reason, for us it will be development mode), then they can do it.